docs/general/gotchas.md

```diff
@@ -21,3 +26,51 @@ import 'file?name=[name].[ext]!../img/yourimg.png';
 ```
 
 Then webpack will correctly transfer the image to the build folder.
+
+## Load reducers optimistically<a name="optimistic-reducers"></a>
+
+If you have some `containers` that aren't _route specific_ and that should be available throughout the app, like a `NavigationBar`, you need to add their respective reducers to the root reducer with the help of `combineReducers`.
+
```

Comment:
Restructure for clarity:

If you have `containers` that should be available throughout the app, like a `NavigationBar` (they aren't _route specific_), you need to add their respective reducers to the root reducer with the help of `combineReducers`.